File: py/fib_l.py
#!/usr/bin/env python3

def main():
    print ((
        (lambda s: s (s))
        (lambda s: lambda n:
          1 if n == 0 or n == 1
          else (s (s))(n - 1) + (s (s))(n - 2)
        )
      ) (33))

main()

File: js/fib_l.js
#!/usr/bin/env jsc

print (
    (function (s) {
        return s (s)
      }
    )
    (function (s) {
        return function (n) {
          if (n == 0 || n == 1) return 1
          else return (s (s))(n - 1) + (s (s))(n - 2)
        }
      }
    )
    (33)
)

